### DISCUSSION TITLE
下载错误：cancel at NeedDlEpsKeyList.oneVideoSelectBestSub

### DISCUSSION THREAD

**Discussion by Ciddmao** [ChineseSubFinder/ChineseSubFinder#550] [Q&A]
使用群晖中矿神的CFS 0.43.1 Lite，电影和电视剧经过了Nastools的处理，电影的字幕下载正常，但电视剧不能下载。详情中看到错误信息：cancel at NeedDlEpsKeyList.oneVideoSelectBestSub
TMDB API连接正常，字幕源API也可用，EMBY也连接正常。

**Comment by baozaolaowu** [comment]
我也遇到了

**Comment by allanpk716** [comment]
需要具体到一个任务，然后反馈这个任务的日志。

**Comment by looperww** [comment]
[WARNING]: 2023-10-10 03:57:39 - cancel Downloader.QueueDownloader()

**Comment by allanpk716** [comment]
> [WARNING]: 2023-10-10 03:57:39 - cancel Downloader.QueueDownloader()

这个问题原因有几种可能：
1. 一个下载任务触发超时
2. 停止了“守护者进程”

**Reply by 516534615** [reply]
守护者进程是在哪里打开呀

**Reply by Yinghai75** [reply]
我也是同样的出错信息。
1. 不是超时，超时要300秒，我是添加任务后立刻看日志就有这信息
2. 守护者进程是开着的

**Comment by zhoutianjun127** [comment]
请问这个问题解决了吗，我新装的也这样

**Comment by DeSireFire** [comment]
![image](https://github.com/ChineseSubFinder/ChineseSubFinder/assets/18726905/fd08883d-82f4-4adb-9afc-dda7c4fc580a)
我使用的是v0.55.3 Lite docker.
![image](https://github.com/ChineseSubFinder/ChineseSubFinder/assets/18726905/552c2a2d-113b-44f5-9511-8f1443ebbe34)
出现了同样的问题。
难道是只有lite版本才会这样？是否有full的docker版本。

**Reply by BeliefanX** [reply]
我新装的，也遇到这个问题了。。。

**Reply by xiNeRoar** [reply]
我也遇到了...整個目錄的影片都是同樣這個ERROR 查看過權限明明沒有問題

**Comment by xiNeRoar** [comment]
請問這個問題有解決方法嗎? 兩年了都找不到辦法解決...

**Reply by gulifeng8** [reply]
电视剧使用了2级目录分类造成，在nastool关闭二级分类即可解决

**Reply by xiNeRoar** [reply]
> 电视剧使用了2级目录分类造成，在nastool关闭二级分类即可解决
謝謝 但我使用的是SONARR 不知道該如何解決 請問可以截圖看一下你正確的目錄層級嗎?

**Comment by gulifeng8** [comment]
同样的问题
